etc/init.d/lncm-post

```diff
@@ -163,6 +163,12 @@ start() {
         /bin/cp -f /home/lncm/nginx/mime.types /media/important/nginx/mime.types  
     fi
 
+    echo "Download index.html"
+    if [ ! -f /home/lncm/public_html/index.html]; then
+      cd /home/lncm/public_html
+      wget https://raw.githubusercontent.com/lncm/invoicer-ui/master/dist/index.html
```

Comment:
We need to change invoicer-ui first to not use "Food4Thought" in the name but have it standard as "LNCM"